StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/OrderingRules/UsingCodeFixProvider.UsingsSorter.cs

```diff
@@ -435,10 +435,7 @@ private bool IsSeparatedSystemUsing(UsingDirectiveSyntax syntax)
                     return false;
                 }
 
-                var namespaceTypeName = namespaceSymbol.ToDisplayString(FullNamespaceDisplayFormat);
-                var firstPart = namespaceTypeName.Split('.')[0];
-
-                return string.Equals(SystemUsingDirectiveIdentifier, firstPart, StringComparison.Ordinal);
+                return syntax.IsSystemUsingDirective();
```

Comment:
This change is incorrect. The original code was using the symbol info to get a fully expanded namespace in the using statement, something that cannot easily be done with only the syntax information.

Reply:
Correct, I misunderstood the original intent here. It might be worth adding a clarifying comment here.

Reply:
Correct, I misunderstood the original intent here. It might be worth adding a clarifying comment here.